podcast/the-changelog-352.md

```diff
@@ -20,7 +20,7 @@ A lot of things that we promoted and pushed were still very brand new at the tim
 
 Some things like that changed, I certainly think for the better, where advice that we gave out, things that we were pushing, we were promoting, have become commonplace, have become widely accepted... So that's kind of heartening, on the one hand.
 
-**Dave Thomas:** And the other thing to remember is that back when this was first written ('98-'99), that was before the snowboard meeting that kicked off the Agile movement \[unintelligible 00:06:08.15\] We were part of that, but it hadn't happened yet, and a lot of the vocabulary that has come to become common nowadays was formed by that movement. We've had the opportunity to simplify some explanations in light of the fact that stuff that we were talking about has now become commonplace as part of the various different agile ways of doing things.
+**Dave Thomas:** And the other thing to remember is that back when this was first written ('98-'99), that was before the snowboard meeting that kicked off the Agile movement for one of the better world. We were part of that, but it hadn't happened yet, and a lot of the vocabulary that has come to become common nowadays was formed by that movement. We've had the opportunity to simplify some explanations in light of the fact that stuff that we were talking about has now become commonplace as part of the various different agile ways of doing things.
```

Comment:
I think this is the phrase used:
```suggestion
**Dave Thomas:** And the other thing to remember is that back when this was first written ('98-'99), that was before the snowboard meeting that kicked off the Agile movement for want of a better word. We were part of that, but it hadn't happened yet, and a lot of the vocabulary that has come to become common nowadays was formed by that movement. We've had the opportunity to simplify some explanations in light of the fact that stuff that we were talking about has now become commonplace as part of the various different agile ways of doing things.
```

Reply:
Sorry @rob729 I should have been more clear:
> for want of a better word
Looks like I was able to update your branch 👍